tests/widgets/test_widget.py

```diff
@@ -0,0 +1,86 @@
+from napari import Viewer
```

Comment:
Maybe rename this `test_napari_imagej.py`?

Reply:
Or at least make this name the same as the file currently named `widgets/napari_imagej.py`.

Reply:
Renamed...in some commit. I don't remember which 😅